pkg/apis/scheduling/v1alpha1/jobschedulingpreference_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// JobSchedulingPreferenceSpec defines the desired state of JobSchedulingPreference
+type JobSchedulingPreferenceSpec struct {
+	// Specifies the maximum desired number of pods this FederatedJob should
+	// run at any given time.
+	// Parallelism specified in the spec for target job template will be
+	// discarded/overridden when job scheduling preferences are specified.
+	// // TODO(irfanurrehman): If needed fix up a strategy about creating jobs
+	// into federated clusters with parallelism as 0. Right now we do not
+	// create jobs in those clusters which have 0 weightage in ClusterWeights.
+	TotalParallelism int32 `json:"totalParallellism"`
```

Comment:
The json tag has an extra L in parallelism here. I Ctrl-F for this but only see it in generated code, so this should be the only place to change.

Reply:
Updated!